src/InputOutput/VTK/writevtk.jl

```diff
@@ -69,11 +69,20 @@ function writevtk(
     auxfieldnames;
     number_sample_points = 0,
 )
+    md = dg.modeldata
     vgeo = dg.grid.vgeo
     device = array_device(Q)
     (h_vgeo, h_Q, h_aux) =
         device isa CPU ? (vgeo, Q.data, state_auxiliary.data) :
         (Array(vgeo), Array(Q), Array(state_auxiliary))
+
+    rnglo = -1
+    rnghi =  1
+    if haskey(md,:mymodeldata)
+     rnglo = Main.get_parm(md.mymodeldata,:my_vtk_range_lo,rnglo)
+     rnghi = Main.get_parm(md.mymodeldata,:my_vtk_range_hi,rnghi)
+    end
```

Comment:
Perhaps a comment might be useful here to explain what "problem" this piece of code is trying to solve?

To summarize the user interface that this design is implementing, if one wants to use the "rnglo" and "rnghi" parameters in VTK output, then one must configure their `modeldata` object to have the fields

```
modeldata.mymodeldata.my_vtk_range_low
```

and

```
modeldata.mymodeldata.my_vtk_range_hi
```

I'd just like to point out that it's probably unsustainable to keep trying to use `modeldata` to hold parameters for VTK output. Is there no object called, say, `VTKOutputWriter` that might be a good place to keep track and interface with these kinds of parameters?

I'm also a bit confused about `Main.get_parm`. Does this mean that a user needs to define the function `get_parm` in the global scope for this functionality to work?

Reply:
@glwagner my_vtk_range_low etc... is sort of just a specific case of a general pattern. It seems likely there are always going to be things where someone wants to tweak a setting, but the data structures don't expose a convenient short cut. This is meant to be a short cut that provides a convenient pattern for getting experimental things done quickly and succinctly that can help with knowing how to evolve API/interfaces etc... 

get_parm could be in a module. Its in Main. at moment just to get the discussion going. If we think this is something that would be useful to have, at least for some workflows, then it can go in a module. 

So this PR is somewhat about deciding if we want to use ocean/dev to experiment with some tools to support rapid shortcuts for configuring ad hoc stuff that colors outside the lines, or if we want to try and get everyone to only color within opinionated interface lines - and leave it to the after market to look after the unofficial add ons?

Reply:
I don't draw a distinction between experimenting and development. There are no lines to color outside of! I am simply trying to provide feedback on this new feature.

I don't know much about the code, which limits my ability to review this contribution. Perhaps this is the best way to achieve your goals. I also don't really understand the problem that's being solved. It looks like your parameter changes `ξdst`. But since I don't know what `ξdst` is, or what "range low" might refer to, I'm pretty much stuck at trying to help further.

More generally it looks like there is a function `writevtk` but no object that represents VTK output. This is presumably the main problem. Since there's no object to carry output specification, extending the code to provide new features re: output is difficult, and multiple dispatch cannot be used to modularize the functionality of `writevtk`. This is probably the main issue --- extending the functionality of the code should not require editing the source code as is done in this PR, but rather should involve defining new functions in your script that extend or change the behavior of existing ones. 

I'd be up for taking a broader look at how VTK output is specified, and designing an abstraction that has the flexibility you need to be productive. To start this discussion, I think an issue is appropriate so that we can review the existing design. If you'd rather just merge this and move on, that's fine too.